templates/etc/ssh/sshd_config.j2

```diff
@@ -66,6 +66,10 @@ MACs {{ sshd_tpl_macs | join(",") }}
 # Privilege Separation is turned on for security
 UsePrivilegeSeparation {{ sshd_privilege_separation }}
 
+{% if sshd_custom_options|d() %}
+# Custom options
+{{ sshd_custom_options }}
+
```

Comment:
Shouldn't this be basically at the top, just after `ansible_managed` to be most effective?